### ISSUE TITLE
Support netstandard library instead of portable

### ISSUE THREAD

**Issue by lvermeulen** [fluentscheduler/FluentScheduler#105]
Since netstandard 1.x is the next generation of portable, I am submitting a PR that gets rid of the Library/PortableLibrary and organizes the project into the typical netcore src/test folders. Targets are net45 and netstandard 1.3 (which gives you net46).

Here's the [pull request](https://github.com/fluentscheduler/FluentScheduler/pull/104)

**Comment by tallesl** [comment]
I updated the project to support netstandard 1.4 instead (f4d608a), Visual Studio offers a handy way to do that:

![](https://cloud.githubusercontent.com/assets/3655047/19841706/a64610c6-9ef6-11e6-91bc-fdb678a3e357.png)

I ended up deciding not to merge your pr, I'm sorry about that.
There's a lot going on there (over 2000 lines changes on 98 files) and some things I'm not quite fond of (like build scripts or how you changed the root directories).

Next time try to discuss the issue before doing the pr and making the pr small as possible, I hope you understand why I'm not merging it.
